src/MonoMod.UnitTest/HookGen/HookGenRunTest.cs

```diff
@@ -63,6 +64,7 @@ public class HookGenRunTest : TestBase {
 }
 
 namespace MonoMod.UnitTest.HookGenTrash.Other {
+    [SuppressMessage("Performance", "CA1812:Avoid uninstantiated internal classes", Justification="Empty stub class.")]
```

Comment:
Ditto about the spaces, though the justifications here are short enough to stay on the same line.